scipy/fft/tests/test_real_transforms.py

```diff
@@ -197,9 +197,10 @@ def test_orthogonalize_noop(func, type, norm, xp):
                   cpu_only=True)
 @pytest.mark.parametrize("norm", ["backward", "ortho", "forward"])
 def test_orthogonalize_dct1(norm, xp):
-    x = xp.asarray(np.random.rand(100))
+    x_np = np.random.rand(100)
+    x = xp.asarray(x_np)
 
-    x2 = xp_copy(x, xp=xp)
+    x2 = xp.asarray(x_np.copy())
```

Comment:
could you remind me why `xp_copy` doesn't work for dask?

Reply:
I think ``xp_copy`` does an ``asarray(..., copy=True)`` to make a copy, but dask silently ignores the ``copy`` flag in its ``asarray`` implementation.

Reply:
hmm, this may be for older Dask only? https://github.com/data-apis/array-api-compat/pull/211 seemed to suggest that copies are now always made even when `copy=None`

Reply:
This is a test-specific issue where xp is the naked dask module. The wrapper returned by `array_namespace` has a functioning copy=True.